reference-architecture/gce-ansible/playbooks/openshift-install.yaml

```diff
@@ -30,6 +28,8 @@
         - service: master http proxy
           port: 8080/tcp
 
+- include: openshift-setup.yaml
+
```

Comment:
@smarterclayton creating the proxy instance service before including openshift-setup.yaml allows for the master lbs to be functional, and prevents issues with node provisioning and other api-related tasks (router, registry, etc).